src/pages/services/aem-cf-console-admin/api/index.md

```diff
@@ -152,39 +190,77 @@ import { register } from "@adobe/uix-guest";
-| ----- | ---- | -------- | ----------- |
-| id | `string` | ✔️    | **Must be unique** across all extensions. Consider adding vendor prefix to this field. |
-| label | `string` | ✔️    | Button label that will be visible on UI |
-| icon | `string` |     | An icon field accepts workflow icon code from @spectrum-icons library - https://spectrum.adobe.com/page/icons/ |
+**Button API**
+
+| Field    | Type                                                                        | Required | Description                                                                                                 |
+|----------|-----------------------------------------------------------------------------| ------ |-------------------------------------------------------------------------------------------------------------|
+| id       | `string`                                                                    | ✔️  | **Must be unique** across all extensions. Consider adding vendor prefix to this field                       |
+| label    | `string`                                                                    | ✔️  | Button label that will be visible on UI                                                                     |
+| icon     | `string`                                                                    |    | An icon field accepts workflow icon code from @spectrum-icons library - https://spectrum.adobe.com/page/icons/ |
+| subItems | `array`                                                                     |    | A lsit with sub menu items                                                                                  |
+| onClick  | `callback(selections): void`                                                          |  ✔️ | A callback for a button `onClick` event. Receives list of content fragments selected in the list            |
+
+**Sub menu item API**
+
+| Field    | Type                         | Required | Description                                                                                                  |
+|----------|------------------------------| ------ |--------------------------------------------------------------------------------------------------------------|
+| id       | `string`                     | ✔️  | **Must be unique** across the current button sub menu                                                        |
+| label    | `string`                     | ✔️  | Button label that will be visible on UI                                                                      |
+| icon     | `string`                     |    | An icon field accepts workflow icon code from @spectrum-icons library - https://spectrum.adobe.com/page/icons/ |
+| onClick  | `callback(selections): void` |  ✔️ | A callback for a button `onClick` event. Receives list of content fragments selected in the list             |
+
+**ContentFragment API**
+
+The structure details can be found in `@aem-sites/headless-sdk`
```

Comment:
`Structure details` words are confusing to me. Please explain in the docs how this related to the other parts of this docs. Also, this package seems to be private and not visible to our users. Should we have a copy of interface here?